### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Good-George28** [community/community#133002] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Discussions

### Body

I really need to know more about Github.com

**Comment by knarrff** [comment]
Hi

Here you go: https://github.com/about. Does this answer your question? :)

**Comment by romhenri** [comment]
What you need?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Good-George28 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and include as many relevant details as possible. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
